backup/README.md

```diff
@@ -18,25 +18,24 @@ pip3 install --user -r requirements.txt
 
 ## Setup
 
-Before using the backup pluging it has to be initialized once. The following
-command will create a `backup.lock` file lightning directoy that stores the
-internal state and makes sure that not two or more instances are using the same
-backup.
+Before the backup plugin can be used it has to be initialized once. The following
+command will create a `backup.lock` in the lightning directory that stores the
+internal state, and which makes sure no instances are using the same backup.
 
 ```bash
 ./backup-cli init ~/.lightning/bitcoin file:///mnt/external/location
 ```
 
 Notes:
- - Make sure to adjust your `~/.lightning/bitcoin` directory to your needs.
+ - Make sure to adjust the `~/.lightning/bitcoin` directory to your needs.
```

Comment:
To be fair I'm not sure what is meant here with adjusting the directory.

Reply:
I added a bit more context, hopefully this clarifies it a bit.